### ISSUE TITLE
Error detected while processing helpme.vim

### ISSUE THREAD

**Issue by duc-mt** [leftbones/helpme-vim#4]
---
name: 🐜 Bug report
about: helpme.vim and undefined variable 🔧
---

Error detected while processing helpme.vim

Please note that I have not made any changes to the helpme-vim plugin folder, the error comes by default.

### Version Information
* Vi IMproved 8.2

### Expected behaviour
I expect to open a file with Vim and enter the command `:HelpMe` to open the menu.

### Actual behaviour
When I open a file in with Vim using `vim file_name`, the command prompt outputs:

```text
Error detected while processing /path/to/folder/helpme-vim/plugin/helpme.vim:
line   25:
E696: Missing comma in List: "" "press 'q' to close" ]
line   39:
E121: Undefined variable: g:HelpMeItems
```

When I ignore the error by pressing the Enter key, the file is opened in Vim.
Then I enter the command `:HelpMe`, the command prompt, again, outputs:

```text
Error detected while processing function <SNR>30_HelpMePopup:
line    1:
E121: Undefined variable: g:HelpMeItems
E116: Invalid arguments for function popup_dialog(g:HelpMeItems, #{ title: g:HelpMeWindowTitle, minwidth: g:HelpMeWidth, minheight: g:HelpMeHeight, border:[], filter: "HelpMeFilter", })
```

**Comment by ShiveshM** [comment]
`g:HelpMeItems` needs to be defined before you load the package

**Comment by duc-mt** [comment]
I think that will only fix the undefined variable error.
The other error is where it says:

```text
Error detected while processing /path/to/folder/helpme-vim/plugin/helpme.vim:
line   25:
E696: Missing comma in List: "" "press 'q' to close" ]
```

**Comment by leftbones** [comment]
@ShiveshM Actually you can load HelpMe! with all of your other plugins and define `g:HelpMeItems` anywhere in your `.vimrc`, the plugin checks for your items when Vim is loaded, and if it's not there, it'll display a default list with some instructions on how to add items.

@tanducmai Ah that's a bug, I missed a comma in the default list I mentioned above, and since you don't have items defined in your `.vimrc` it's trying to populate with a non-working list. Just make sure you use commas in your list and it'll be fine! I'll push a fix to it in a minute.

**Comment by leftbones** [comment]
Fixed in 0268df6
